src/cartesian/Bar.tsx

```diff
@@ -556,14 +556,15 @@ function BarImpl(props: Props) {
 
   const barSettings: BarSettings = useMemo(
     (): BarSettings => ({
+      id: props.id,
```

Comment:
This should be the autogenerated ID, not just the optional props variant - I'll fix that in next PR.